presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestSourcePartitionedScheduler.java

```diff
@@ -122,6 +125,12 @@ public void setUp()
         finalizerService.start();
     }
 
+    @BeforeMethod
+    public void setUpX()
```

Comment:
you can just name it as `setUp` and drop the X

Reply:
There already is a `@BeforeClass` method called `setUp`.